build.sbt

```diff
@@ -201,11 +205,18 @@ lazy val coreTest = project
     testkit
   )
 
+lazy val appCommons = project
+  .in(file("app/app-commons"))
```

Comment:
so in my opinion this shouldn't be in the `app/` as it isn't a standalone application like our server/cli/dlc client.